redhat-cop/casl-ansible
# Dns selinux

#### What does this PR do?
Updated to handle SELinux correctly for DNS servers

#### How should this be manually tested?
Install a dns-server and ensure that SELinux doesn't cause problems when adding new records with nsupdate

#### Is there a relevant Issue open for this?
N/A

#### Who would you like to review this?
cc: @redhat-cop/casl
